### ISSUE TITLE
Nominatim does not interpolate house number from sourrounding building

### ISSUE THREAD

**Issue by Juanjo-MC** [osm-search/Nominatim#2683]
## What did you search for?

[B Event](https://nominatim.openstreetmap.org/ui/search.html?q=B+Event)

## What result did you get?

B Event, Avenue de la Brise - Brieslaan, Mutsaard, Laeken - Laken, City of Brussels, Brussels-Capital, 1020, Belgium Caterer

## What result did you expect?

B Event, **_36_**, Avenue de la Brise - Brieslaan, Mutsaard, Laeken - Laken, City of Brussels, Brussels-Capital, 1020, Belgium Caterer

## Further details

Housenumber is missing from address for amenities mapped as nodes  and located inside a building way. It seems to me that, in the past,  Nominatim was returning this information, interpolated from the surrounding building.

**Comment by lonvia** [comment]
Yeah, I recently noticed that too. The information is there, it's just not output.
